lib/iris/tests/integration/test_trajectory.py

```diff
@@ -329,8 +333,12 @@ def test_hybrid_height(self):
         # Put a lazy array into the cube so we can test deferred loading.
         cube.data = as_lazy_data(cube.data)
 
+        # Use opionated grid-latitudes to avoid the issue of platform
+        # specific behaviour within SciPy cKDTree choosing a different
+        # equi-distant nearest neighbour point when there are multiple
+        # valid candidates.
         traj = (
-            ("grid_latitude", [20.5, 21.5, 22.5, 23.5]),
+            ("grid_latitude", [20.4, 21.6, 22.6, 23.6]),
```

Comment:
We could leave this as-is... but I'm again erring on the side of doing the obvious for clarity and robustness sake.